### PR TITLE
Fedora rsyslog_cron_logging rule

### PR DESCRIPTION
**Pull request by mildass**
#### Description:
Add rsyslog_cron_logging rule for Fedora profile and add tests for this rule.
Also fix minor problems that I found in OVAL/remediation.

### CODE REVIEW

File: linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_cron_logging/oval/shared.xml [hunk 1]
```diff
@@ -4,7 +4,8 @@
     <metadata>
       <title>Verify Cron is Logging to Rsyslog</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_rhel</platform>
+	<platform>multi_platform_fedora</platform>
```
Comment (jan-cerny) [inline linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_cron_logging/oval/shared.xml]: inconsistent indents

File: tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/set_cron_logging.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Function to add cron logging line to specific file
+# Expects argument:
+# file:		Configuration file, where we want cron logging set
+#
+function set_cron_logging {
+	local file=$1
+
+	if grep -s '^[[:space:]]*cron\.\*' $file; then
```
Comment (jan-cerny) [inline tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/set_cron_logging.sh]: Shouldn't be the variable quoted?

File: tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/cron_set_rsyslog_conf.pass.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_nist-800-171-cui, xccdf_org.ssgproject.content_profile_ospp
+. set_cron_logging.sh
+
+RSYSLOG_CONF='/etc/rsyslog.conf'
+RSYSLOG_D_FILES='/etc/rsyslog.d/*'
+
+# Ensure that rsyslog.conf exist and rsyslog.d folder doesn't contain any file with cron.*
```
Comment (jan-cerny) [inline tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/cron_set_rsyslog_conf.pass.sh]: exists

File: tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/wrong_log_rsyslog_conf.fail.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_nist-800-171-cui, xccdf_org.ssgproject.content_profile_ospp
+. set_cron_logging.sh
+
+RSYSLOG_CONF='/etc/rsyslog.conf'
+RSYSLOG_D_FILES='/etc/rsyslog.d/*'
+
+# Ensure that rsyslog.conf exists and rsyslog.d folder doesn't contains any file with cron.*
```
Comment (jan-cerny) [inline tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/wrong_log_rsyslog_conf.fail.sh]: doesn't contain

File: linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_cron_logging/bash/shared.sh [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+
+if ! grep -s "^\s*cron\.\*\s*/var/log/cron$" /etc/rsyslog.conf /etc/rsyslog.d/*.conf; then
+	mkdir -p /etc/rsyslog.d
+	echo "cron.*                                                  /var/log/cron" >> /etc/rsyslog.d/cron.conf
```
Comment (dahaic) [inline linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_cron_logging/bash/shared.sh]: Unless there is a point in having awkwardly long whitespace, remove it please.

File: tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/set_cron_logging.sh (hunk 2 above)
Comment (mildass) [reply tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/set_cron_logging.sh]: It isn't necessary here, but for perfection it's better with it so I'll add it.

**Pull request merged**